src/main/java/com/nina/modelling/shoes/KidShoe.java

```diff
@@ -0,0 +1,34 @@
+package com.nina.modelling.shoes;
+
+public class KidShoe extends Shoe {
+    private String category;
+    private String accessory;
+    private String type;
+
+    public KidShoe(String category, String accessory, String type) {
+        this.category = category;
+        this.accessory = accessory;
+        this.type = type;
+    }
+
+    public KidShoe(String category) {
```

Comment:
contructors with less params should be placed first

Reply:
Now it"s first